app/http/controllers/AuthController.ts

```diff
@@ -0,0 +1,32 @@
+import e, { NextFunction, Request, Response } from "express";
+import { AuthService } from "../../services/AuthService";
+import axios from "axios";
+import NotFoundException from "../../exceptions/NotFoundException";
+
+
+export const AuthController = {
+
+/**
+	 * Handles the default request on /
+	 * 
+	 * @param req {Request} Express request object
+	 * @param res {Response} Express response object
+	 * @param next {NextFunction} Express NextFunction (used for middleware)
+	 */
+
+    
+ async login(req: Request, res: Response, next: NextFunction) {
+
+    axios.post("http://hslinux:38383/api/v1/auth", {
```

Comment:
Put the codes in authService not authController. The AuthController should only call function, pass in arguments and respond to the request